### ISSUE TITLE
Can't update location on Weather Meter

### ISSUE THREAD

**Issue by oobedoob** [SilverAzide/Gadgets#96]
It may be something on my end or not. Location Search doesn't appear to work, entering in City, State and pressing Enter or entering Zip Code and pressing Enter results in no results also when trying to enter longitude/latitude as soon as I click out of the boxes it reverts to the original entries. So at this point, I'm permanently stuck with the weather for Manhattan.

**Comment by SilverAzide** [comment]
Dammit.  Looks like openstreetmap.org is finally blocking Rainmeter like they have been threatening to do.

The OpenStreetMap admins recently contacted the Rainmeter devs complaining that improperly written skins are hammering their website with repeated requests, and threatening to cut off access if it isn't stopped.  There is nothing the Rainmeter devs can do about this because it is crappily-written skins that are doing it.  So it looks like OpenStreetMap finally pulled the trigger.

You can fix this behavior easily enough.  Open the Location Search skin like normal, but right-click the skin and select "Edit skin" from the menu.  Find the measure `[MeasureLocations]`.  Add the following line to the measure:
```ini
UserAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36 Edg/92.0.902.67"
```
Save the file, and right-click the skin and select "Refresh skin".  Finally, enter your city or address and press <Enter>.  It should now work.

Thanks for the notification!!!!

**Comment by SilverAzide** [comment]
This will be addressed in a patch release, out shortly.

**Comment by oobedoob** [comment]
> Dammit. Looks like openstreetmap.org is finally blocking Rainmeter like they have been threatening to do.
> 
> The OpenStreetMap admins recently contacted the Rainmeter devs complaining that improperly written skins are hammering their website with repeated requests, and threatening to cut off access if it isn't stopped. There is nothing the Rainmeter devs can do about this because it is crappily-written skins that are doing it. So it looks like OpenStreetMap finally pulled the trigger.
> 
> You can fix this behavior easily enough. Open the Location Search skin like normal, but right-click the skin and select "Edit skin" from the menu. Find the measure `[MeasureLocations]`. Add the following line to the measure:
> 
> ```ini
> UserAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36 Edg/92.0.902.67"
> ```
> 
> Save the file, and right-click the skin and select "Refresh skin". Finally, enter your city or address and press Enter. It should now work.
> 
> Thanks for the notification!!!!

Editing the location skin fixed it. Thanks!

**Comment by SilverAzide** [comment]
Fixed in v7.6.1.
